.gitattributes

```diff
@@ -1 +1,2 @@
 * text=auto
+.devcontainer/*.sh text eol=lf
```

Comment:
Isn't `lf` the default?

Reply:
"auto" makes the checkout/staging compatible with the underlying platform (thanks to the clean/smudge mechanism), while always storing files in git with lf line endings.

For instance, on Windows, with "auto" configuration, files are stored within git with lf line endings, but checking them out store them with crlf line endings on the disk.

"auto" is the sensible choice for 99.9% setups.

The added line overrides this settings by special casing the shell scripts and requesting git to always have them bear lf line endings on disk, independently of the underlying platform.